compiler-core/src/type_/printer.rs

```diff
@@ -181,6 +181,12 @@ impl Names {
         _ = self.type_variables.insert(id, local_alias.clone());
     }
 
+    pub fn map_new_variable(&mut self, old_id: u64, new_id: u64) {
```

Comment:
Could you document what this does please? 🙏 

I was a bit confused by the word "map" as in my mind that means a function like `list.map`. Is there a name that someone like me with less context would find extra clear?

Reply:
With pleasure!

renamed the function to `reassign_type_variable_alias` which is much more explicit, also added a small comment on top describing what it does. Hope it helps!